proc_scripts/integral_w_error.py

```diff
@@ -0,0 +1,35 @@
+from pyspecdata import *
+from .integrate_limits import integrate_limits
+import numpy as np
+def select_pathway(s,pathway):
+    retval = s
+    for k,v in pathway.items():
+        retval = retval[k,v]
+    return retval    
+def integral_w_errors(self,sig_path,error_path, indirect='vd', direct='t2'):
+    """need a docstring!!!"""
+     frq_slice = integrate_limits(self)
+     s = self[direct:frq_slice]
+     t = self.getaxis(direct)
+     dt = t[1]-t[0]
```

Comment:
somehow, everything here and below is indented 5 spaces.  fix after reviewing comments + changes